spec/automation/unit/method_validation/orchestration_retirement_spec.rb

```diff
@@ -82,7 +83,7 @@
 
     it "deletes stack from vmdb" do
       ws
-      OrchestrationStack.where(:id => stack.id).first.should be_nil
+      expect(OrchestrationStack.where(:id => stack.id).first).to be_nil
```

Comment:
this should probably be  `expect(OrchestrationStack.exists?(:id => stack.id)).to be_falsey`, in a followup PR